packages/vx-network/src/DefaultLink.tsx

```diff
@@ -1,12 +1,8 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import { LinkProvidedProps } from './types';
 
-DefaultLink.propTypes = {
-  link: PropTypes.object,
-};
-
-export default function DefaultLink({ link }) {
-  return (
+export default function DefaultLink({ link }: LinkProvidedProps<any>) {
```

Comment:
can it be `LinkProvidedProps<Link<DefaultNode>>`?

Reply:
I tried this at first, but got TS errors in the `Links` component when setting `linkComponent = DefautlLink` along the lines of `Link<DefaultNode>` is not assignable to `Link` 😭 

wasn't sure of a better solution than this 👎